### DISCUSSION TITLE
Errors installing Emscripten during setup steps on a Windows machine

### DISCUSSION THREAD

**Discussion by andyhe958** [climateinteractive/SDEverywhere#358] [Q&A] [answered]
Hi everyone,

I have been trying to run SDEverywhere from command prompt on a Windows machine but came across an error below when installing Emscripten. It suggests Python is not found, but in fact I do have Python 3.11 installed.

![Error message 1](https://github.com/climateinteractive/SDEverywhere/assets/146044517/2c6aea19-0e85-4544-bd01-75001671e94a)

Then I went on to disable the app alias for Python as suggested, but I got a second error, which seems to contradict with the first error:

![Error message 2](https://github.com/climateinteractive/SDEverywhere/assets/146044517/58217792-05c1-4088-981b-b7ff357827ee)

I have since managed to install Emscripten separately, not as part of the SDEverywhere setup steps, and it was installed successfully. I think I would post this here to help others potentially in a similar situation, and to help with further configuration of this tool for Windows.

**Comment by chrispcampbell** [comment]
Hi @andyhe958, thanks for reporting this.  I recall when I wrote this step that it was a little sensitive to which method Python was installed on Windows (there are a couple options, and maybe the one I had used worked while others do not).  It also depends on whether the `python3` alias is on the Windows `PATH` at the time (I think some installs only come with `python`).  Anyway, I'll open an issue for this so that we can get it fixed.

Please feel free to post any other questions/comments here on the discussion board as you get up and running with SDEverywhere, feedback is always welcome and helpful!

**Reply by chrispcampbell** [reply, marked as answer]
For the record, this is being tracked in issue #359.  There are some additional details in that issue, and I hope to fix the install script on Windows so that it avoids this issue with Python.  I will post updates in that issue.
